package.json

```diff
@@ -36,8 +36,13 @@
     "pegjs": "0.7.0",
     "request": "2.79.0",
     "underscore": "1.8.3",
-    "winston": "2.3.0",
-    "nodeunit": "0.10.2",
+    "winston": "2.3.0"
+  },
+  "devDependencies": {
```

Comment:
hmm looks like I didn't test this like I thought I did. I still have a bit to learn about npm.